lte/gateway/c/session_manager/LocalSessionManagerHandler.h

```diff
@@ -121,38 +121,68 @@ class LocalSessionManagerHandlerImpl : public LocalSessionManagerHandler {
   void add_session_to_directory_record(
       const std::string& imsi, const std::string& session_id);
 
+  /**
+   * handle_create_session_cwf handles a sequence of actions needed for the
+   * RATType=WLAN case. It is responsible for responding to the original
+   * LocalCreateSession request.
+   * If there is an existing session for the IMSI that is ACTIVE, we will
+   * simply update its SessionConfig with the new context. In this case, we will
+   * NOT send a CreateSession request into FeG/PolicyDB.
+   * Otherwise, we will go through the procedure of creating a new context.
+   * @param session_map - SessionMap that contains all sessions with IMSI
+   * @param request - the original request coming in from AAA
+   * @param sid - newly created SessionID
+   * @param cfg - newly created SessionConfig from the LocalCreateSessionRequest
+   * @param cb - callback needed to respond to the original
+   * LocalCreateSessionRequest
+   */
+  void handle_create_session_cwf(
+      SessionMap& session_map, const LocalCreateSessionRequest& request,
+      const std::string& sid, SessionConfig cfg,
+      std::function<void(Status, LocalCreateSessionResponse)> cb);
+
+  /**
+   * handle_create_session_lte handles a sequence of actions needed for the
```

Comment:
I may be missing something, but where are those functions implemented?

Reply:
In `LocalSessionManagerHandler.cpp` :) It should be in this PR

Reply:
ohh for some reason github decided not to show it to be too big